tools/gdb/README.md

```diff
@@ -0,0 +1,13 @@
+# Custom gdb scripts
+Custom gdb scripts for gdb debugging and autoloading of these scripts.
+individual scripts are located in the `gdb_scripts/` and the top level directory is reserved for loader scripts `load_gdb_scripts.py`.
+
+### Usage
+To loads scripts from within gdb call `source /path/to/load_gdb_scripts.py`.
+If you have compiled WiredTiger with the `-DHAVE_SHARED=1` flag then these scripts 
+will be auto-loaded when gdb is opened.
+
+### Creating new scripts
+Scripts can be written in either python ([hazard_pointers.py](./gdb_scripts/hazard_pointers.py)) or scheme ([dump_row_int.gdb](./gdb_scripts/dump_row_int.gdb)). If you create a new python script it is recommended you follow the style in `hazard_pointers.py` by creating a class that extends `gdb.Command`.
```

Comment:
I see you mention style here as well, I guess we may need to determine what the "correct" style is.

Reply:
Are we talking python code style here or making sure we use classes inheriting from `gdb.Command`?
If it's the latter I think we should encourage a single way to structure scripts from the get-go.

Reply:
Python code style, they should all inherit gdb.Command

Reply:
SGTM. README text updated in [131d4c4](https://github.com/wiredtiger/wiredtiger/pull/9801/commits/131d4c41328f96f5d5e8e2b861cf2a453e75a58e)